addons/interaction/functions/fnc_takeLauncher.sqf

```diff
@@ -0,0 +1,52 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Timi007. Cplhardcore
+ * Pass launcher to another unit.
+ *
+ * Arguments:
+ * 0: Unit that passes the launcher <OBJECT>
+ * 1: Unit to pass the launcher to <OBJECT>
+ * 2: Launcher classname <STRING>
+ * 3: Play passing animation <BOOL> (default: true)
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_player, _target, "launch_NLAW_F"] call ace_interaction_fnc_passLauncher
+ *
+ * Public: No
+ */
+params ["_player", "_target", "_launcher", ["_animate", true, [true]]];
+TRACE_4("Take launcher params",_player,_target,_launcher,_animate);
+private _playerName = [_player] call EFUNC(common,getName);
+_displayName = getText (_cfgWeapons >> "displayName");
```

Comment:
_cfgWeapons undefined
_displayName not private

Reply:
Fixed